tools/simulator/libsimulator/lib/runtime/RuntimeCCSImpl.cpp

```diff
@@ -3,7 +3,7 @@
 #include "ConfigParser.h"
 #include "cocostudio/CocoStudio.h"
 #include "ui/UIHelper.h"
-#include "tinyxml2/tinyxml2.h"
```

Comment:
If you don't use prebuilts, since the 'external' folder is present in the include path, it founds the tinyxml2 in the external folder before finding the system one.